app/src/main/java/org/sil/bloom/reader/WiFi/NewBookListenerService.java

```diff
@@ -85,7 +74,7 @@ private void listen(Integer port) throws Exception {
         try {
             JSONObject data = new JSONObject(message);
             String title = data.getString("Title");
-            newBookVersion = data.getString("Version");
+            String newBookVersion = data.getString("Version");
```

Comment:
This is unusual to upper case json property names. https://stackoverflow.com/questions/5543490/json-naming-convention

Is it too late to fix this?